src/handlers/staking_handlers.ts

```diff
@@ -41,6 +43,23 @@ export class StakingHandlers {
 
         res.status(HttpStatus.OK).send(response);
     }
+    public async getStakingEpochNAsync(req: express.Request, res: express.Response): Promise<void> {
+        // optional query string to include fees
+        schemaUtils.validateSchema(req.query, schemas.stakingEpochRequestSchema as any);
+        const isWithFees = req.query.withFees ? req.query.withFees === 'true' : false;
+
+        const n = Number(req.params.n);
+
+        let response: Epoch | EpochWithFees;
+        if (isWithFees) {
+            const [epoch] = await Promise.all([this._stakingDataService.getEpochNWithFeesAsync(n)]);
```

Comment:
NIT: The `Promise.all` kinda superfluous since you're are only awaiting 1 promise.
```suggestion
            const epoch = await this._stakingDataService.getEpochNWithFeesAsync(n);
```